e2e/utils.go

```diff
@@ -41,7 +41,8 @@ const (
 	appKey   = "app"
 	appLabel = "write-data-in-pod"
 
-	noError = ""
+	noError    = ""
+	trueString = "true"
```

Comment:
I understand that some go linter complains about the re-use of `"true"`. Not sure this is a very nice approach to silence the warning though.

As the new `trueString` is only used for setting `staticVolume`, you can call the constant `isStaticVolume`?

Or use `strconv.FormatBool(true)` instead of `"true"`?

Reply:
Except for this new nit, I have nothing to complain about or need to see improvements. :+1:

Reply:
This is because due to `goconst` linter. as "true" is already using in 2 places and we are adding one more `true` How about having a helper function to return string value and reuse the same in other places also. or else will leave that as it is forgotten.

Reply:
> How about having a helper function to return string value and reuse the same in other places also. or else will leave that as it is forgotten.

Added `strconv.FormatBool(true)` at those places too, I don't think we need any other helper function to just return `"true"` other than this.